doc/api/crypto.md

```diff
@@ -3432,6 +3474,52 @@ const n = crypto.randomInt(1, 7);
 console.log(`The dice rolled: ${n}`);
 ```
 
+### `crypto.randomPrime(size[, options[, callback]])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `size` {number} The size (in bytes) of the prime to generate.
```

Comment:
Measuring prime sizes in bytes is confusing. For example, 7 is technically a one-byte prime, but only a 3-bit prime. I'd strongly prefer using bits here.